environment.yml

```diff
@@ -8,3 +8,6 @@ dependencies:
   - pandas=1.2
   - pytest=6.2
   - python=3.8
+  - pip=21.1.1
+  - pip:
+     - git+https://github.com/annaritz/graphspace-python@fa9bb4b0adc54de50a956daade82db48a8030f64
```

Comment:
Yes, conda-forge sounds like a better change here. We don't want to have too many library dependencies, since the point of this is to keep it streamlined for the user. 

Another move would be to dockerize `graphspace-python` so the current PyPI version works. This could also be a  bigger change, if we want to handle encrypted passwords (related to Issue #14).

Reply:
I created #30 for this.  I'll work on it after we merge this pull request.

What are the pros and cons of running graphspace-python in Docker versus installing it in our conda environment?  Some downstream analysis, like parameter advising, would be best in Docker because it depends on C++ code and system-specific installation.  For Python packages that are compatible the the rest of our conda environment, we could include them in the environment by default.

Reply:
Let's keep it as a Python dependency for now, and see if the change I made gets merged and a new version gets released. If it is slow to be integrated, then we can revisit this.